### PR TITLE
add test-utils feature to polkadot-node-core-pvf

### PR DESCRIPTION
**Pull request by Jpserrat**
Companion for https://github.com/paritytech/polkadot/pull/7538

### CODE REVIEW

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/cumulus/-/jobs/3355425

**Comment by Jpserrat** [comment]
@mrcnski @slumber Cumulus companion created.

**Review by mrcnski — APPROVED** [review]

**Review by slumber — APPROVED** [review]

**Comment by mrcnski** [comment]
Agh! I'll try to fix your companion. Luckily companions won't be a thing anymore soon.

**Comment by ggwpez** [comment]
Green now. Can we merge to unblock the CI?

File: Cargo.lock [hunk 1]
```diff
@@ -5372,30 +5351,24 @@ dependencies = [
...
-name = "itoa"
-version = "1.0.4"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "4217ad341ebadf8d8e724e264f13e593e0648f5b3e94b3896a5df283be015ecc"
+checksum = "af150ab688ff2122fcef229be89cb50dd66af9e01a4ff320cc137eecc9bacc38"
 
 [[package]]
 name = "jobserver"
-version = "0.1.24"
+version = "0.1.26"
```
Comment (ggwpez) [inline Cargo.lock:5369]: Why is this updating so many deps?

**Review by ggwpez — CHANGES_REQUESTED** [review]
I think something with the update is wrong with the cargo update. It should be -+300 lines, not 2200.

**Comment by mrcnski** [comment]
@ggwpez What command should I run? I'm getting

```
$ cargo update
    Updating crates.io index
    Updating git repository `https://github.com/paritytech/substrate`
    Updating git repository `https://github.com/paritytech/polkadot`
error: no matching package named `pallet-multisig` found
location searched: https://github.com/paritytech/substrate?branch=master
required by package `asset-hub-kusama-runtime v0.9.420 (/Users/parity/Repos/github.com/paritytech/cumulus-2/parachains/runtimes/assets/asset-hub-kusama)`
```

**Comment by ggwpez** [comment]
Need this first, there is some upstream problem in Substrate master https://github.com/paritytech/substrate/pull/14763

**Comment by mrcnski** [comment]
Got it thanks. Hopefully this is the last companion I ever have to merge. 😄

**Review by ggwpez — APPROVED** [review]
I reverted the last updates, this is roughly how it should look like; not +-2200.

image

**Comment by mrcnski** [comment]
Thanks @ggwpez! I had tried reverting Cargo.lock then `cargo update`, but guess I messed up something.

**Pull request merged**